supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/models/GcpBatchWorkflowPaths.scala

```diff
@@ -46,6 +46,7 @@ case class GcpBatchWorkflowPaths(workflowDescriptor: BackendWorkflowDescriptor,
 
   private val workflowOptions: WorkflowOptions = workflowDescriptor.workflowOptions
 
+  // TODO: Do we actually need this? it seems to be used only by tests
```

Comment:
I wonder if anyone can comment on this, this method seems unnecessary because its only used by the tests, the same occurs in PAPI.

Reply:
Looks like completely dead code in Batch, fine to remove IMO. IntelliJ doesn't even find any references in tests.

Reply:
Yes, I skipped porting these tests but I wanted to double check before removing the code.

Thanks for looking into it.